exist-core/src/main/java/org/exist/xquery/functions/util/Eval.java

```diff
@@ -283,13 +283,11 @@ private Sequence doEval(final XQueryContext evalContext, final Sequence contextS
         //set module load path
         if (Type.subTypeOf(expr.getType(), Type.ANY_URI)) {
             String uri = null;
-            final Object key = querySource.getKey();
 
-            if (key instanceof XmldbURI) {
-                uri = XmldbURI.EMBEDDED_SERVER_URI.append(((XmldbURI) key).removeLastSegment()).toString();
-//          } else if (key instanceof URL) {
-//              TODO: uri = ((URL) key).getParent()
-            } else if (key instanceof String && querySource instanceof FileSource) {
+            if (querySource instanceof DBSource) {
```

Comment:
DBsource can be remote too right?

Reply:
I am not quite sure what you are asking? But no... DBSource is always local. I have not changed the previous behaviour here.

Reply:
ok thnx; just asking.